low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -163,8 +168,52 @@ def get_cpu_info(self):
         ]
         return cpu_data
 
+    def get_json_formatted_reading(self, reading):
```

Comment:
rename this function to something denoting ipmi response formatter

Reply:
Done.